### PR TITLE
install-deps.sh: Update python requirements for FreeBSD

### PR DESCRIPTION
**Pull request by wjwithagen**
signed-off-by: Willem Jan Withagen <wjw@digiware.nl>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: install-deps.sh [hunk 1]
```diff
@@ -212,7 +212,9 @@ if [ x$(uname)x = xFreeBSDx ]; then
         java/junit \
         lang/python \
         lang/python27 \
+        lang/python36 \
         devel/py-pip \
+	devel/py-flake8 \
```
Comment (tchaikov) [inline install-deps.sh]: could you use spaces for indent? just to be consistent with other packages in this command.

**Comment by tchaikov** [comment]
could you use `install-deps.sh` instead of `build` as the prefix?

File: install-deps.sh (hunk 1 above)
Comment (wjwithagen) [reply install-deps.sh]: @tchaikov 
Fix other ommissions as well

**Comment by wjwithagen** [comment]
@tchaikov 
Fixed commit description as well.

**Pull request merged**